pkg/cmd/cli/cmd/create.go

```diff
@@ -54,8 +52,8 @@ func CmdCreate(f *clientcmd.Factory, reader io.Reader, out io.Writer) *cobra.Com
 		},
 	}
 
-	cmd.Flags().Int("masters", 1, "Numbers of workers in spark cluster")
-	cmd.Flags().Int("workers", 1, "Numbers of workers in spark cluster")
+	cmd.Flags().Int("masters", 0, "Numbers of workers in spark cluster")
```

Comment:
This has to do with how oshinko-core handles cluster configurations.  The master and worker counts are part of a config object that looks like this:

    type ClusterConfig struct {
	    MasterCount int
	    WorkerCount int
	    Name string
	    SparkMasterConfig string
	    SparkWorkerConfig string
    }

In this object, Name is the name of a stored cluster config.
If a stored cluster config is named, that configuration is read first by oshinko-core. It may contain values for all of the other fields of a ClusterConfig, and all stored configs have defaults of WorkerCount = 1 and MasterCount = 1 if they are not specified (0 == not specified)

If no named config is specified, an internal default config is used with WorkerCount = 1.

Then, if any of the other fields are explicitly set in the config object passed to create, those values overwrite the values read from the stored config. Again, 0 == unspecified for counts, and "" == unspecified for string fields.

So if we make the default value for --workers and --masters be 0, the clusters will pick up the defaults from oshinko or from the stored config.  If we leave them set to 1 explicitly, they will overwrite the default settings or a value from a stored config.